constants/networks/BSC.json

```diff
@@ -0,0 +1,5 @@
+{
+  "url": "https://bsc-dataseed.binance.org/",
```

Comment:
I remember when I did the DNS for the BSC, those publicly available nodes didn't work properly, very often they failed. We had to register with a vedor to get an archive node. As those are private information, we would need to use environment variables here.

If this file is used for tests (I suppose it is), then we need to setup Github secrets:
https://docs.github.com/en/rest/actions/secrets?apiVersion=2022-11-28